app/src/main/java/com/gun0912/tedpermissiondemo/NormalActivity.java

```diff
@@ -5,46 +5,50 @@
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 import android.widget.Toast;
+
 import com.gun0912.tedpermission.PermissionListener;
 import com.gun0912.tedpermission.TedPermission;
+
 import java.util.ArrayList;
 
 /**
  * Created by TedPark on 16. 2. 21..
  */
 public class NormalActivity extends AppCompatActivity {
 
-  @Override
-  protected void onCreate(@Nullable Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
+    @Override
```

Comment:
There is no reason to change indent.
Please revert it!

Reply:
My bad! Reverted it in last commit.